### DISCUSSION TITLE
"What is Neurodesk" FAQ is great!

### DISCUSSION THREAD

**Discussion by civier** [neurodesk/neurodesk.github.io-hugo#146] [Ideas]
Just wanted to say that the "What is Neurodesk" section (https://neurodesk.github.io/docs/faq/#what-is-neurodesk) is great and worth linking to it from the main page maybe.
If you explain me how to edit the main page, I can try to do it myself.

**Comment by stebo85** [comment]
I added the FAQ section to the landing page

**Comment by Ianyliu** [comment]
the new link is https://www.neurodesk.org/docs/overview/faq/#what-is-neurodesk
